kipoi/postprocessing/variant_effects/mutation_map.py

```diff
@@ -0,0 +1,683 @@
+import copy
+import itertools
+import logging
+import os
+import tempfile
+import h5py
+
+import numpy as np
+import pandas as pd
+import six
+from tqdm import tqdm
+from vcf.model import _Record, _Substitution
+import matplotlib.pyplot as plt
```

Comment:
matplotlib should be in `setup.py` as well if we import it here. 

I'm thinking whether it would make sense to make matplotlib as an optional dependency. In concise, I sometimes had trouble with matplotlib installation.

Reply:
I'd pack maptplotlib into https://github.com/kipoi/kipoi/blob/master/setup.py#L57 and import it in the function itself